helm/prometheus-meta-operator/templates/secret.yaml

```diff
@@ -8,5 +8,10 @@ metadata:
 stringData:
   secret.yml: |
     service:
+      prometheus:
+        remoteWrite:
+          basicAuth:
+            username: 10755
```

Comment:
Can we also have the username templated ? Nice to have, I am ok if we can't.

Reply:
Right now, not really, we can add that in the helm values but it is not in the installations repo. You can add a PR for it though :)

Reply:
Ok fine